onnxruntime/test/onnx/main.cc

```diff
@@ -354,6 +354,28 @@ int real_main(int argc, char* argv[]) {
   broken_tests["fp16_inception_v1"] = "need to adjust the per_sample_tolerance: 0.002";
 #endif
 
+#ifdef _WIN32
+  broken_tests["resnet50"] = "failed: type mismatch";
```

Comment:
These tests currently fail in the pipeline for x86 build. However, if run the test on individual model, each test pass. So, they are more of problem in the test framework. Need to be fixed (TODO)

```
PS C:\Users\prs\work_projects\ort_x86> .\build\Windows\RelWithDebInfo\RelWithDebInfo\onnx_test_runner -j 1 -n resnet101v2 -n resnet50 -n tf_resnet_v1_50 -n tf_resnet_v2_50 -n tf_resnet_v1_101 -n tf_resnet_v1_152 -n tf_resnet_v2_101 -n tf_resnet_v2_152 -n vgg19 -n tf_nasnet_large -n tf_pnasnet_large -n resnet101v2 -n resnet152v2 -n tf_inception_resnet_v2 -n tf_inception_v4 -n tf_inception_v3 -n resnet50v2 C:\Users\prs\Downloads\20190130\ C:\Users\prs\work_projects\onnxruntime\cmake\external\onnx\onnx\backend\test\data

2019-03-05 17:49:46.7401428 [W:onnxruntime:Default, graph.cc:2193 onnxruntime::Graph::CleanUnusedInitializers] gpu_0/imagenet1k_blobs_queue_f22e83c9-22cd-4a8b-a66d-113af6b832b4_0 exists in this graph's initializers but it is not used by any node

2019-03-05 17:49:46.8097771 [W:onnxruntime:resnet50, session_state_initializer.cc:499 onnxruntime::SaveInputOutputNamesToNodeMapping] Graph input with name gpu_0/imagenet1k_blobs_queue_f22e83c9-22cd-4a8b-a66d-113af6b832b4_0 is not associated with a node.

2019-03-05 17:50:22.5466323 [W:onnxruntime:Default, graph.cc:2193 onnxruntime::Graph::CleanUnusedInitializers] gpu_0/imagenet1k_blobs_queue_f22e83c9-22cd-4a8b-a66d-113af6b832b4_0 exists in this graph's initializers but it is not used by any node

2019-03-05 17:50:22.6130785 [W:onnxruntime:resnet50, session_state_initializer.cc:499 onnxruntime::SaveInputOutputNamesToNodeMapping] Graph input with name gpu_0/imagenet1k_blobs_queue_f22e83c9-22cd-4a8b-a66d-113af6b832b4_0 is not associated with a node.

result:

        Models: 40

        Total test cases: 363

                Succeeded: 363

                Not implemented: 0

                Failed: 0

        Stats by Operator type:

                Not implemented(0):

                Failed:

Failed Test Cases:
```

Reply:
@pranavsharma

Reply:
It worth to investigate.